tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ts

```diff
@@ -64,29 +64,41 @@ export class ScalarCardDataTable {
     startPointIndex: number,
     endPointIndex: number,
     smoothed: boolean = false
-  ): number {
+  ): ScalarCardPoint {
     let minValue = this.maybeSmoothedValue(points[startPointIndex], smoothed);
+    let minValuePoint = points[startPointIndex];
     for (let i = startPointIndex; i <= endPointIndex; i++) {
       if (minValue > this.maybeSmoothedValue(points[i], smoothed)) {
         minValue = this.maybeSmoothedValue(points[i], smoothed);
+        minValuePoint = points[i];
       }
     }
-    return minValue;
+    return minValuePoint;
   }
 
   getMaxValueInRange(
```

Comment:
Should you change the function names to getMaxPointInRange and getMinPointInRange?